release_notes/41-release-notes.adoc

```diff
@@ -0,0 +1,428 @@
+| GA
+| DEP
+| REM
+
+| `roxctl scanner generate` flag `offline-mode` (flag only)
+| GA
+| DEP
+| DEP
+
+| `/v1/report` endpoint
+| GA
+| DEP
+| DEP
+
+| `/v1/imagecves/suppress` and `/v1/imagecves/unsuppress`
+| GA
+| GA
+| DEP
+
+| `/v1/serviceaccounts` endpoint
+| GA
+| DEP
+| DEP
+
+| Vulnerability Management 1.0: Image CVEs, Image Components, Images, Deployments, and Namespaces.
+| GA
+| GA
+| DEP
+
+| Custom Security Context Constraints (SCCs): `stackrox-collector`,`stackrox-admission-control`, and `stackrox-sensor`
+| GA
+| GA
+| DEP
+
+| `Vulnerability Management Approver` permission
+| GA
+| GA
+| DEP
+
+| `Vulnerability Management Requester` permission
+| GA
+| GA
+| DEP
+
+| `Vulnerability Report Creator` permission
+| GA
+| GA
+| DEP
+
+| `Vulnerability Reports` and `Policy` permission
+| DEP
+| DEP
+| REM
+
+| `Role` resource
+| DEP
+| DEP
+| REM
+
+| `Scope Manager` system role and permission set
+| DEP
+| DEP
+| REM
+
+| `ROX_FORCE_LOCAL_IMAGE_SCANNING` environment variable
+| DEP
+| DEP
+| REM
+
+|====
+
+
+[id="deprecated-features_{context}"]
+=== Deprecated features
+
+The following section provides additional information about deprecated features listed in the preceding table.
+
+* Avoid using the following sections in Vulnerability Management 1.0: Image CVEs, Image Components, Images, Deployments, and Namespaces. They are deprecated and it is anticipated that these sections will be removed in a future release. Instead, use Vulnerability Management 2.0 to manage workload vulnerabilities.
+
+* Ensure that you do not use Custom security context constraints (SCCs) such as `stackrox-collector`,`stackrox-admission-control`, and `stackrox-sensor` for workloads other than {product-title-short}. These SCCs are deprecated and it is anticipated that they will be removed in a future release.
+
+* The default `Vulnerability Management Approver` permission set is deprecated. Use the `Vulnerability Management Admin` permission instead. Existing roles that use `Vulnerability Management Approver` will be updated to use `Vulnerability Management Admin` when the permission set is removed in the future.
+
+* The default `Vulnerability Management Requester` permission set is deprecated. Use the `Vulnerability Management Consumer` permission instead. Existing roles that use `Vulnerability Management Requester` will be updated to use `Vulnerability Management Consumer` when the permission set is removed in the future.
+
+* The default `Vulnerability Report Creator` permission set is deprecated. Use the `Vulnerability Management Admin` permission instead. Existing roles that use `Vulnerability Report Creator` will be updated to use `Vulnerability Management Admin` when the permission set is removed in the future.
+
+[id="removed-features_{context}"]
+=== Removed features
+
+The following section provides additional information about removed features listed in the preceding table.
+
+[id="role-resource_{context}"]
+==== `Role` resource
```

Comment:
No backticks in headings - fix here and in next few headings

Reply:
fixed all instances